packages/teleport/src/Kubes/Kubes.tsx

```diff
@@ -62,23 +70,48 @@ export function Kubes(props: State) {
           </ButtonPrimary>
         )}
       </FeatureHeader>
-      <Flex flex="0 0 auto" mb={4}>
-        <InputSearch mr="3" onChange={e => setSearchValue(e)} />
-      </Flex>
       {attempt.status === 'failed' && <Danger>{attempt.statusText}</Danger>}
       {attempt.status === 'processing' && (
         <Box textAlign="center" m={10}>
           <Indicator />
         </Box>
       )}
-      {attempt.status === 'success' && (
-        <KubeList
-          kubes={kubes}
-          username={username}
-          authType={authType}
-          searchValue={searchValue}
+      {hasKubes && (
+        <>
+          <Flex flex="0 0 auto" mb={4}>
+            <InputSearch mr="3" onChange={setSearchValue} />
+          </Flex>
+          <KubeList
+            kubes={kubes}
+            username={username}
+            authType={authType}
+            searchValue={searchValue}
+            setSearchValue={setSearchValue}
+          />
+        </>
+      )}
+      {isEmpty && (
+        <Empty
+          clusterId={clusterId}
+          isLeafCluster={isLeafCluster}
+          isEnterprise={isEnterprise}
+          canCreate={canCreate}
+          onClick={() => window.open(DOC_URL)}
+          emptyStateInfo={emptyStateInfo}
         />
       )}
     </FeatureBox>
   );
 }
+
+const emptyStateInfo: EmptyStateInfo = {
+  title: 'ADD YOUR FIRST KUBERNETES CLUSTER',
+  description:
+    'Teleport Application Access provides secure access to internal applications without the need for a VPN but with the audibility and control of Teleport.',
```

Comment:
Fast, secure access to Kubernetes clusters. Follow [the documentation](https://goteleport.com/docs/kubernetes-access/guides/multiple-clusters/) to connect your first cluster.